xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/htmlheader.vm

```diff
@@ -148,10 +148,14 @@
 #end
 ## These elements will be closed in the htmlfooter.vm file
 ## We put a space before each class added in #if, to avoid losing them because of velocity line stripping
+## TODO: the classes should be refactored to use a array of classes where individal classes are happened on by one and
+## joined with space at the end
+## Note: when showLeftPanels and showRightPanels are both equals to 0, it is intentional of hideleft and hideright to
+## be concatenated without space.
 <$bodyTagName id="body" class="skin-flamingo wiki-${xcontext.database} ${bodyAction}body panel-left-width-${leftPanelsWidth} panel-right-width-${rightPanelsWidth} drawer drawer--right ${preferenceUnderlining}
   #if("$!doc.space" != "") space-${escapetool.xml($doc.space.replaceAll(' ', '_'))}#end
   #if($showLeftPanels == "0") hideleft#end
-  #if($showRightPanels == "0") hideright#end
+  #if($showRightPanels == "0")hideright#end
```

Comment:
I don't have the impression that this fixed anything. The left column is still shown on the login form.

Reply:
Or, maybe even clearer, `#if($showLeftPanels == "0") hideleft#end#if($showRightPanels == "0")hideright#end` both in the same line

Reply:
I did remove it in https://github.com/xwiki/xwiki-platform/pull/3074/commits/3c1fcc7505bd411258bc57c135a6f4464c930f43 but somehow it got added back before the merge :/